.github/workflows/status-ci-cd.yml

```diff
@@ -28,29 +26,3 @@ jobs:
       commit_resources: true
     secrets:
       access_token: ${{ secrets.COMMIT_TOKEN }}
-  validate-website-reference:
```

Comment:
Question, unclear if it is blocking: we need to determine how we will generate schemas in these pipelines even if they are not committed inline anymore, and that means adapting metaschema-xslt, in this PR branch or another.

Reply:
I'm not sure if this has been addressed or not, or if it is still a potential issue -- @nikitawootten-nist ?

Reply:
If we are planning on removing auto-comitting actions, we should probably do the following:
1. Streamline the schema and converter generation to be as simple to invoke as possible (another makefile?)
2. Modify the existing action to build the schemas and converters without committing for PRs
3. Also make the action upload schema artifacts on release

For existing releases we'll have to manually upload the correct artifacts (which is simple but a bit tedious, thankfully we only have a few releases so it shouldn't be too painful)